prime-router/docs/observability/azure-events.md

```diff
@@ -254,3 +254,34 @@ customEvents
 | summarize count() by conditionDisplay
 | order by count_
 ```
+
+### List of items that were not routed where patient lived in state X
+```kql
+customEvents
+| where name == "ITEM_NOT_ROUTED"
+| extend params = parse_json(tostring(customDimensions.params))
+| where params.bundleDigest.patientState contains "X"
+```
+
+### List of items that failed a filter where patient lived in state X
```

Comment:
What if we also join this with the metadata from the item accepted event? That is something Engagement would be doing specifically so might be good to add here.